lib/omnibus/builder.rb

```diff
@@ -149,6 +149,7 @@ def patch(options = {})
       source = options.delete(:source)
       plevel = options.delete(:plevel) || 1
       target = options.delete(:target)
+      forward = options.delete(:forward) || false
```

Comment:
nitpick: could you add a description of the parameter in the method's inline docs above?